.github/workflows/master.yml

```diff
@@ -719,7 +797,80 @@ jobs:
           echo "*** $numPass Passed ***"
           if [ "$numFail" -gt 0 ];then echo "*** $numFail Failed ***";exit 2;fi
           if [ "$numPass" -eq 0 ];then echo "*** 0 Passed! ***";exit 2;fi
-          if [ "$numPass" -lt 12 ];then echo "*** Failed to pass at least 6 ! ***";exit 2;fi
+          if [ "$numPass" -lt 12 ];then echo "*** Failed to pass at least 12 ! ***";exit 2;fi
+          if [ "$numPass" -eq 0 ];then echo "*** Passed! ***";exit 0;fi
```

Comment:
Is this line just in case we skip everything? I don't think we'll ever get here because of the previous line.

Reply:
this happens when test times out without any failure, like 2 test would be passing and it timed out (witout any failure) so it would give u a false sense of success